vcpkg.json

```diff
@@ -34,7 +34,6 @@
     "ableton-link",
     "benchmark",
     "chromaprint",
-    "fdk-aac",
```

Comment:
We need to keep this for broadcasting and recording. 
It is the also the RH Legal approved patent-free version.

Reply:
So [this VCPKG version of `fdk-aac`](https://vcpkg.io/en/package/fdk-aac) is approved by RH but not the `ffmpeg` on e? On what grounds? VCPKG's `fdk-aac` use the FreeDesktop stripped src?

Reply:
See: https://github.com/mixxxdj/vcpkg/commit/d306ab43c77c28d2f1d002093db6d4386cd7b333
We did the same move we are now doing for FFmpeg a wile ago. 

I can imagine that the native aac encoder has not been legally approved because we have no list of pathents that it has implemented. For the fdk-aac dekoder this list exists and a legal approval is less tedious on that list compared to reading and understanding the source and checking for possible violated patents.